src/test/semaphore_compat.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Ceph - scalable distributed file system
```

Comment:
Not sure whether this is the correct license to specify. @neha-ojha ?

Reply:
Please advise here, I just copied the header from a file nearby

Reply:
Until there's a more qualified answer, I'd suggest removing the license paragraph altogether